.github/workflows/dynamoMSbuild.yml

```diff
@@ -18,7 +18,8 @@ jobs:
       run: |
         echo "***Continue with the build, Good luck developer!***"
         cd "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\"
-           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln
+           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln /t:Restore
+           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln /t:Build
```

Comment:
Since on the line where it looks for the exe, is using the debug folder, I've left it on that way, @mjkkirschner  could tell us more about why on this case the debug (default) configuration was applied

Reply:
Shouldn't it be release?

Reply:
@mjkkirschner Can you help us solve this question since the original is using Debug configuration? If you want me to switch to Release configuration I can do the change

Reply:
I think either is fine, though I believe @QilongTang added the 2019 windows build job - unless we start using this for smoke tests I don't think it matters.

Reply:
Either is fine. I added this test only to verify that Dynamo builds on latest Windows VM/MS toolset provided by Github. Do not meant to target debug or release